### PR TITLE
RK-14558 - Example instructions

### PR DESCRIPTION
**Pull request by lirook**
Added some instruction text on using the Ingress example.

### CODE REVIEW

File: charts/rookout-hybrid/README.md [hunk 1]
```diff
@@ -12,31 +12,39 @@ For more information:
...
 helm repo add rookout https://helm-charts.rookout.com
 helm repo update
 ```
 
-### Full installation - Controller + Datastore
+### Installation with Ingress
+
+In most cases, the Datastore requires an Ingress for user access, which acts as a TLS termination proxy.
+
+[This yaml file](https://github.com/Rookout/helm-charts/tree/master/charts/rookout-hybrid/examples/ingress_example.yaml) shows an example configuration for an Nginx Ingress controller and Let's Encrypt. To use it, modify the contents of the example yaml file to match your configuration, and then run:
```
Comment (LiranLast) [inline charts/rookout-hybrid/README.md:28]: Let's document this file :)

File: charts/rookout-hybrid/README.md [hunk 2]
```diff
@@ -12,31 +12,39 @@ For more information:
...
 
 ```commandline
 helm repo add rookout https://helm-charts.rookout.com
 helm repo update
 ```
 
-### Full installation - Controller + Datastore
+### Installation with Ingress
+
+In most cases, the Datastore requires an Ingress for user access, which acts as a TLS termination proxy.
```
Comment (LiranLast) [inline charts/rookout-hybrid/README.md:26]: Let's create slimmer copy :)

**Comment by LiranLast** [comment]
@naomirookout can you take it from here?

**Review by gosharo — APPROVED** [review]

**Pull request merged**